nltk/classify/rte_classify.py

```diff
@@ -70,8 +70,8 @@ def __init__(self, rtepair, stop=True, lemmatize=False):
         self.hyp_words = set(self.hyp_tokens)
 
         if lemmatize:
-            self.text_words = set(lemmatize(token) for token in self.text_tokens)
```

Comment:
ah I see. based on the queries results I mentioned above I think it can be argued that `RTEFeatureExtractor` constructor and `lemmatize()` are both equally public and unused - but I think you're right that it'd probably make more sense to rename the parameter.

Maybe something like `use_lemmatize`? I'll wait to hear from @stevenbird on what it'd prefer to call it